### DISCUSSION TITLE
Don't alter global settings from gameplay load screen

### DISCUSSION THREAD

**Discussion by ticotaco72** [ppy/osu#13044] [Ideas]
**Describe the new feature:** there should be toggle in game settings, if editing settings in gameplay loading should be saved as global or as per beatmap or per beatmapset

**Comment by peppy** [comment]
We do not save settings per beatmap. That said, we may want to consider doing what stable does, which is to not modify the global setting when it is changed at the loading screen. Unsure about that.

**Comment by Morilli** [comment]
Would certainly like to see this implemented, as I often want to adjust values for a specific map only.
